cookie-notice.md

```diff
@@ -0,0 +1,93 @@
+
+**Do not track:** Some browsers include the ability to transmit "Do Not Track" or "DNT" signals. Although we do our best to honor the privacy preferences of our users, we are not able to respond to Do Not Track signals from your browser at this time. We take privacy and meaningful choice seriously and will make efforts to continue to monitor developments around DNT browser technology and the implementation of a standard.
+
+  
+
+### 6\. EDTECH ANALYTICS
+
+Clever has developed a proprietary analytics service ("Edtech Analytics") that collects analytics and statistical analysis related to the use of educational websites, applications and services by students and teachers. While this service does not use cookie and similar technologies, it does collect data using a browser extension. This includes information such as URL, time spent on page, use of third party educational websites and applications, unique device ID, browser version, and similar data. Clever provides this service for our customers, who are responsible for installing and configuring the service. You can find out more about Edtech Analytics [here](https://support.clever.com/hc/s/articles/360039420391?language=en_US).
+
+  
+
+### 7\. HOW TO CONTACT US
+
+If you have any questions about this Cookie Notice, we recommend contacting us by visiting our [Help Center](https://support.clever.com/).
+
+Additionally, Clever can be reached:
+
+*   By Email: [trust@clever.com](mailto:trust@clever.com)
+*   By Mail: Clever Inc, ATTN: Legal Department, 1263 Mission St, San Francisco CA 94103
+*   By Phone: 1-800-521-6516
+
+If we use cookies and similar technologies to collect personal information, or to collect information that becomes personal information if we combine it with other information, we will process such personal information in accordance with our [Privacy Policy](/trust/privacy/policy).
+
+  
+
+### 8\. CHANGES TO OUR COOKIE NOTICE
+
+Clever may modify or update this Cookie Notice from time to time so you should review this page periodically. [Click here](https://github.com/Clever/policies/releases) to see all changes to this Cookie Notice. You can see when this Cookie Notice was last updated by checking the "last modified" date indicated below.
```

Comment:
Update this to match the cookie notice html.

Reply:
updated the final changes section in both notices